<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Preset Positions</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Current Presets</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>X</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Y</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Z</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>A:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="line_A_X"><property name="readOnly"><bool>true</bool></property></widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="line_A_Y"><property name="readOnly"><bool>true</bool></property></widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="line_A_Z"><property name="readOnly"><bool>true</bool></property></widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>B:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="line_B_X"><property name="readOnly"><bool>true</bool></property></widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="line_B_Y"><property name="readOnly"><bool>true</bool></property></widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="line_B_Z"><property name="readOnly"><bool>true</bool></property></widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>C:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="line_C_X"><property name="readOnly"><bool>true</bool></property></widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="line_C_Y"><property name="readOnly"><bool>true</bool></property></widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="line_C_Z"><property name="readOnly"><bool>true</bool></property></widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>D:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="line_D_X"><property name="readOnly"><bool>true</bool></property></widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="line_D_Y"><property name="readOnly"><bool>true</bool></property></widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="line_D_Z"><property name="readOnly"><bool>true</bool></property></widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Setting</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>ForSet:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="input_Set_X"><property name="placeholderText"><string>X Value</string></property></widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="input_Set_Y"><property name="placeholderText"><string>Y Value</string></property></widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="input_Set_Z"><property name="placeholderText"><string>Z Value</string></property></widget>
      </item>
      <item row="1" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_GetInfo">
          <property name="text">
           <string>Get Info</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="combo_Target">
          <item><property name="text"><string>A</string></property></item>
          <item><property name="text"><string>B</string></property></item>
          <item><property name="text"><string>C</string></property></item>
          <item><property name="text"><string>D</string></property></item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_SetPosition">
          <property name="text">
           <string>Set Position</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>